quiche/src/recovery/bbr2/mod.rs

```diff
@@ -55,7 +55,7 @@ const PACING_MARGIN_PERCENT: f64 = 0.01;
 
 /// A constant specifying the minimum gain value
 /// for calculating the pacing rate that will allow the sending rate to
-/// double each round (4*ln(2) ~=2.77 ) [BBRStartupPacingGain]; used in
+/// double each round (4*ln(2) ~=2.77 ) BBRStartupPacingGain; used in
```

Comment:
This is to fix the "unresolved link to `BBRStartupPacingGain`" warnings that pop up in other PRs